playbooks/roles/tracking_logs_sync/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
@mrgnr Yes, I'm going to create a card for this, and we can address it in the next sprint.